src/main/java/PlayingLinkedStack.java

```diff
@@ -0,0 +1,68 @@
+import java.util.NoSuchElementException;
+
+public class PlayingLinkedStack {
```

Comment:
Insanely minor, it's pretty typical to put playing code into a `playground` package.

Reply:
The only reason I'm not is I worry about the students downloading it and the package throwing them off.